bin/server.js

```diff
@@ -40,4 +42,24 @@ DB.connect(config)
     server.on('mem', function (stats) {
       logger.info('mem', stats)
     })
+    // Publish notifications via simple background loop.
+    if (config.notifications.publishUrl) {
+      var sleepTime
```

Comment:
I don't immediately see why `sleepTime` isn't declared in the publish function's scope?